src/value-converters/README.md

```diff
@@ -1,8 +1,8 @@
 # Value Converters
 
-## anti-xss-html-sanitizer
+## diy-html-sanitiser
 
-Sanitizes HTML against XSS attacks for `innerhtml`.
+Sanitizes HTML against XSS attacks for `innerhtml`, etc. via blacklist and RegEx. Tries to escape all elements / discards nothing.
```

Comment:
Or move into new README.md in subfolder? I'd rather wait until (iff) there is more.